README.md

```diff
@@ -100,11 +100,25 @@ class MyComponent extends Component {
 }
 ```
 
-Options:
+In order to prevent the [unknown props warning](https://facebook.github.io/react/warnings/unknown-prop.html)
```

Comment:
I wouldn't mention this issue that is more user-code specific, I'd rather talk about possible props clash with another library